easybuild/easyblocks/b/boost.py

```diff
@@ -120,6 +123,13 @@ def prepare_step(self, *args, **kwargs):
     def configure_step(self):
         """Configure Boost build using custom tools"""
 
+        # boost_multi_thread is deprecated
+        if self.cfg['boost_multi_thread'] is not None:
+            self.log.deprecated("boost_multi_thread has been deprecated, it has been replaced by tagged_layout. "
```

Comment:
Minor message improvement suggestion (the deprecation part is already printed by default more or less. I think it will look like: "Deprecated in EasyBuild 5.0: boost_multi_thread has been replaced by tagged_layout...")
```suggestion
            self.log.deprecated("boost_multi_thread has been replaced by tagged_layout. "
```